### DISCUSSION TITLE
Install chrome extensions

### DISCUSSION THREAD

**Discussion by StudioCompile** [3kh0/ext-remover#1353] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
my school's chrome extension is not blocked only with admin under my account (have my home account added) and they just have an allowlist. I know about the bookmarklet but they blocked that, it's not blocked in the policy URLs I checked so idk how they blocked it, I also removed all my extensions, and still doesn't work so idk. And extensions can't run code on the chrome.google.com/webstore. Is there any way even on Google Chrome on my home computer synced with the school to install an extension?

**Comment by Charlieis1nsan3** [comment]
try chrome://inspect.

**Reply by Charlieis1nsan3** [reply]
I think?

**Comment by schoolexploitkid** [comment, marked as answer]
its blocked by `developerToolsAvailability`, u cant bypass that

**Reply by ghost** [reply]
how

**Reply by S-PScripts** [reply]
its a policy and you cant change those
